yarl/_url.py

```diff
@@ -521,7 +521,7 @@ def password(self):
         """
         return self._UNQUOTER(self.raw_password)
 
-    @property
+    @cached_property
```

Comment:
Does this mean that the tests need to be updated to access `__wrapped__`?

Reply:
Since the URL objects are immutable we shouldn't have to worry about the value every changing or needing to test `__wrapped__`.  I don't see any of the other tests using `cached_property` doing/needing that.